http/src/main/scala/org/http4s/blaze/http/http2/client/ClientSessionImpl.scala

```diff
@@ -0,0 +1,35 @@
+package org.http4s.blaze.http.http2.client
+
+import java.nio.ByteBuffer
+
+import org.http4s.blaze.http.{Http2ClientSession, HttpRequest}
+import org.http4s.blaze.http.HttpClientSession.{ReleaseableResponse, Status}
+import org.http4s.blaze.http.http2._
+import org.http4s.blaze.pipeline.{Command, LeafBuilder, TailStage}
+import org.log4s.getLogger
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ExecutionContext, Future}
+
+private final class ClientSessionImpl(
+    tailStage: TailStage[ByteBuffer],
+    localSettings: ImmutableHttp2Settings,
+    remoteSettings: MutableHttp2Settings,
+    flowStrategy: FlowStrategy,
+    parentExecutor: ExecutionContext)
+  extends Http2ClientSession {
+
+  override def dispatch(request: HttpRequest): Future[ReleaseableResponse] = ???
```

Comment:
This is for a future PR I assume.

Reply:
Yes, just a stub. Sorry, I'll add a comment to that effect.